modules/sagemaker/sagemaker-studio/stack.py

```diff
@@ -0,0 +1,211 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License").
+#    You may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from typing import Any, List
+
+import aws_cdk as core
+from aws_cdk import Stack
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_lambda as lambda_
+from aws_cdk import aws_sagemaker as sagemaker
+from aws_cdk.aws_lambda_python_alpha import PythonFunction
+from aws_cdk.custom_resources import Provider
+from constructs import Construct
+
+from helper_constructs.networking import Networking
+from helper_constructs.sm_roles import SMRoles
+
+
+class SagemakerStudioStack(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        *,
+        deployment_name: str,
+        module_name: str,
+        vpc_id: str,
+        subnet_ids: List[str],
+        studio_domain_name: str,
+        studio_bucket_name: str,
+        data_science_users: List[str],
+        lead_data_science_users: List[str],
+        app_image_config_name: str,
+        image_name: str,
+        **kwargs: Any,
+    ) -> None:
```

Comment:
you have a return statement below....so this signature doesn't match.  Please remove the return denoted below?